manticore/__main__.py

```diff
@@ -112,7 +117,7 @@ def main():
 
     env = {key: val for key, val in map(lambda env: env[0].split('='), args.env)}
 
-    m = Manticore(args.argv[0], argv=args.argv[1:], env=env, workspace_url=args.workspace,  policy=args.policy, disasm=args.disasm, concrete_start=args.data)
+    m = Manticore(args.argv[0], argv=args.argv[1:], env=env, entry=args.entrysymbol, workspace_url=args.workspace,  policy=args.policy, disasm=args.disasm, concrete_start=args.data)
```

Comment:
There's a disconnect from command line argument naming (`entrysymbol`) to internal naming (`entry`), especially that the TODO indicates future support for entry.

Reply:
I meant that.
Argument parsing should only allow either one.
And we can pass it to subsequent functions which could tell the difference, based on the type (an address would be an unsigned int and a symbol would be a string).
What do you think of this ?